policies/release-requirements.md

```diff
@@ -0,0 +1,91 @@
+Release Requirements Policy
+===========================
+
+The OpenSSL project team creates the following three types of OpenSSL software
+releases:
+
+- [stable] releases
+- [alpha] (pre-)releases
+- [beta] (pre-)releases
+
+This policy defines the requirements on the state of the source tree that
+must be met before the release can be done.
+
+Alpha Pre-releases
+------------------
+
+As this is just a preview release for testing things that have been worked
+on the development branch of the source code the requirements are minimal.
+
+- The CI must pass on tip of the development branch before the release commits
+  were added to the tree.
+
+Beta Pre-releases
+-----------------
+
+The API and ABI should be stable and the source code should be feature complete
+by the first beta pre-release for the stable release. The following release
+requirements apply to beta pre-releases.
+
+- The code is functionaly complete in regards to the particular release
+  objectives as set by OMC and OTC.
+- There is no remaining refactoring required by OMC or OTC for the release.
+- There are no remaining API changes required for the release.
+  _This applies only to beta releases of a major release as API changes
+  are not allowed on minor releases._
+- All untriaged issues must be triaged.
```

Comment:
Untriaged issues that have only just been opened (e.g. in the last 1 or 2 weeks) should perhaps be exempted. Otherwise if someone opens a new issue just before you do the release it would be a blocker.

Reply:
I've added `older than 2 days`. IMO the basic triage should be really quick task in most cases so maybe even `older than 1 day` would be OK.